core/modules/system/config/system.core.json

```diff
@@ -78,5 +78,6 @@
     "user_picture_style": "thumbnail",
     "user_picture_dimensions": "1024x1024",
     "user_picture_file_size": "800",
-    "user_picture_guidelines": ""
-}
\ No newline at end of file
+    "user_picture_guidelines": "",
+    "watchdog_enabled_severity_levels": [0, 1, 2, 3, 4, 5, 6]
```

Comment:
We need an update hook to add this setting to existing sites. This might make it for 1.4.0, but https://github.com/backdrop/backdrop-issues/issues/1832 probably won't, so until then we'll need to make update hooks for changed config.